flake.nix

```diff
@@ -161,9 +161,9 @@
       cheriotRtosSource = pkgs.fetchFromGitHub {
         owner = "lowRISC";
         repo = "CHERIoT-RTOS";
-        rev = "30148bad7ef45026d4e4b754871bf2b396ce2ca8";
+        rev = "444e4965c1dc84414e4959d55e47b0e999c086c9";
```

Comment:
Can you sync this up with the version here: https://github.com/lowRISC/sonata-system/blob/cae088cf016aa3b26fc3b3dfbb2623282e4fdb8d/sw/cheri/CMakeLists.txt#L10

Reply:
If I don't point to the `hyperram` branch as described then the RTOS cannot be built & run in CI - the current Git tag being used in the `CMakeLists.txt` file does not include the hyperram hack.
I've instead updated the commit to sync the CMake CHERIoT RTOS soruce up with this nix change.